labs/vstsextend/sonarcloud/readme.md

```diff
@@ -312,6 +330,70 @@ Open the **Sonar Examples - NetFx** project in the SonarCloud Dashboard.  Under
 
    Users will now be unable to merge the pull request until the Code Quality check is successful, either because all of the issues have been fixed or the the issues have been marked as **confirmed** or **resolved** in SonarCloud.
 
+## Exercise 4: Check the SonarCloud Quality Gate status in a Continuous Deployment scenario (In Preview)
+
+**Disclaimer : This feature is in preview, and may not reflect it's final version. Please look at the notes at the end of this exercise for more information.**
+
+Starting from version 1.8.0 of the SonarCloud extension for Azure DevOps, a pre-deployment gate is available for your release pipeline. It allows you to, before a deployment, check the status of the SonarCloud QualityGate for the artifact you want to deploy.
```

Comment:
"QualityGate" normally appears as separate words in the documentation. However, I see from the screen shots that it appears as one word in the task name and description. Should it be one word or two in the task name?

Reply:
You're right. We should ask Fabrice maybe as well for that.

Reply:
Ok, in the mean time, i redacted all QualityGate to Quality Gate, even in the extension.